redis/redis_test.go

```diff
@@ -0,0 +1,45 @@
+package redis
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/garyburd/redigo/redis"
+)
+
+const testServer = "redis://localhost:6379"
```

Comment:
We should do something about `testServer`, probably remove it.

It's only used in one place now, in the test skip message, which is far away from here, and completely ignored by the `redis.Dial` function call.

Leaving it as is just adds confusion.